matrix/docker/query_runner.py

```diff
@@ -1,9 +1,81 @@
 """Script to pull from sqs and run redshift queries. Will be dockerized."""
+import json
+import os
 
+from matrix.common.aws.sqs_handler import SQSHandler
+from matrix.common.aws.s3_handler import S3Handler
+from matrix.common.aws.batch_handler import BatchHandler
+from matrix.common.aws.redshift_handler import RedshiftHandler
+from matrix.common.config import MatrixInfraConfig
+from matrix.common.logging import Logging
+from matrix.common.request.request_tracker import RequestTracker, Subtask
 
-def main():
-    pass
+logger = Logging.get_logger(__name__)
+
+
+class QueryRunner:
+
+    def __init__(self):
+        self.sqs_handler = SQSHandler()
+        self.s3_handler = S3Handler(os.environ["MATRIX_QUERY_BUCKET"])
+        self.batch_handler = BatchHandler()
+        self.redshift_handler = RedshiftHandler()
+        self.matrix_infra_config = MatrixInfraConfig()
+
+    @property
+    def query_job_q_url(self):
+        return self.matrix_infra_config.query_job_q_url
+
+    @property
+    def query_job_deadletter_q_url(self):
+        return self.matrix_infra_config.query_job_deadletter_q_url
+
+    def run(self, max_loops=None):
```

Comment:
Example use case for `max_loops`?

Reply:
suggest: rename and break into smaller functions for testing and clarity

Reply:
Currently only used by test_run in test_query_runner so that the tests don't run forever. Let me know if you have a better way to handle that

Reply:
Hmmm, I could remove out some of the sqs message parsing. I think the function calls are clean though and tell the story of the run function

Reply:
Sounds good. Yea you're right, it's just the parsing and logging that looks clunky. Np